dashboard/app/views/levels/editors/fields/_help_and_tips.html.haml

```diff
@@ -18,3 +18,7 @@
       .span8
         - @level.reference_links = [''] unless @level.reference_links.try(:present?)
         = text_area_tag 'level[reference_links]', @level.reference_links.join("\r\n"), rows: 6, class: "input-block-level", placeholder: 'Additional Reference Link'
+
+  .field
+    = render partial: 'levels/editors/fields/checkboxes', locals: {f: f, field_name: :ai_tutor_available, description: "AI Tutor Available"}
+  
```

Comment:
nit: add a blank line to the end of this file

Reply:
actually nvm, looks like you did this. Not sure why git is showing the no blank line symbol

Reply:
@molly-moen Do you have an auto-formatter for haml files that you use? Looks like Prettier doesn't cover this type

Reply:
I don't think so :( I don't touch haml often